src/components/stepper/index.js

```diff
@@ -4,7 +4,7 @@ import Button from "../button";
 import "./style.css";
 import { useState } from "react";
 
-const Stepper = ({ header, children, onComplete }) => {
+const Stepper = ({ header, children, onComplete, data }) => {
```

Comment:
This `Stepper` component is a generic component that should be able to be used anywhere in the app. By adding this, you've tightly coupled it to a dataset that relies on a firstName and lastName existing.

Let's take this one back to the drawing board to think about how you might disable / enable the buttons in it in a more reusable way